dat/outfits/unique/scary_paint.xml

```diff
@@ -12,15 +12,6 @@
   <desc_extra>Projects an unsettling visual appearance onto the ship's hull, causing a mild shock to those that lay their eyes upon it. Pirates tend to respect those that go the extra mile just for appearance's sake.</desc_extra>
   <gfx_store>black_cat_doll.webp</gfx_store>
   <limit>fake_transponder</limit>
-  <illegalto>
```

Comment:
Why did you remove illegality? I like the idea of it being a trade-off between pirate love and other faction love.

Reply:
I found after some testing that illegality is not required, since all the factions that check for illegality also do a scan, and anyone that scans you and isn't a pirate will set you as hostile. So this gives the player a choice -- shoot back and lose standings or ignore them (but don't lose standings). I could go either way, but I found it was a bit annoying to lose 2% faction standing at the very end of a journey right before landing.
But, it's up to you and all the other players, as I said, I could go either way, but I think illegality here is a little bit overkill because whoever scans you will start harassing you anyway, which either forces you to respond or run away/tank.